aporia.nim

```diff
@@ -2110,7 +2111,16 @@ proc initsourceViewTabs() =
 
         win.forceScrollToInsert(int32(newTab))
 
-      else: dialogs.error(win.w, "Could not restore file from session, file not found: " & filename)
+      else: missingFilenames.add(filename)
+
+    if len(missingFilenames) > 0:
+      var filenames = ""
+      if len(missingFilenames) > 5:
+        var numNotShown = len(missingFilenames) - 5
+        filenames = missingFilenames[0..4].join("\n") & "\nand " & intToStr(numNotShown) & " more files"
```

Comment:
I like this in general but I dislike that I won't be shown some of the missing files.